src/dashboard-refactor/header/SearchBar/SearchBar.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { PureComponent } from 'react'
+import Margin from 'src/dashboard-refactor/components/Margin'
+import styled from 'styled-components'
+
+import colors from '../../colors'
+import { fonts } from '../../styleConstants'
+
+interface SearchBarProps {
+    handleSearchBoxClick(): void
```

Comment:
I don't know if it's possible in this case, but if this event handler only has one specific reason to exists, it might be better to make the name more specific, like `onSearchBoxFocus()` if you want to allow users of this component to track focus outside of the component.